exercise13_1.py

```diff
@@ -48,5 +55,8 @@
         counter += 1
         if js["results"][0]["address_components"][counter]["types"] == ['country', 'political']:
             print(js["results"][0]["address_components"][counter]["short_name"])
+            break
+        elif counter == len(info)-1:
+            print("No country code")
```

Comment:
I think this can also be placed outside the for loop so we don't need to check the counter each time.